h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
+        model['is_moderator'] = True
```

Comment:
I think given that the `public` property is currently a boolean, I'd lean towards calling this `moderator` rather than `is_moderator`.